AdminUi/src/AdminUi/appsettings.override.json

```diff
@@ -23,9 +23,9 @@
       "GcpPubSubTopicName": "" // only available for Google Cloud Pub/Sub
     },
     "SqlDatabase": {
-      "Provider": "Postgres",
-      "ConnectionString": "User ID=adminui;Password=Passw0rd;Server=localhost;Port=5432;Database=enmeshed;" // postgres
-      //"ConnectionString": "Server=localhost;Database=enmeshed;User Id=AdminUi;Password=Passw0rd;TrustServerCertificate=True"
+      "Provider": "SqlServer",
+      //"ConnectionString": "User ID=adminui;Password=Passw0rd;Server=localhost;Port=5432;Database=enmeshed;" // postgres
```

Comment:
Here the user is all lowercase. In SqlServer it is camelCase. I would prefer to have both of them in camelCase. Otherwise it's hard to remember.